www/src/pages/pt/deployment/docker.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Docker
+description: Deploy com Docker
+layout: ../../../layouts/docs.astro
+lang: pt
+---
+
+Você pode colocar essa stack em contêineres e fazer deploy dela como um único contêiner usando o Docker ou como parte de um grupo de contêineres usando o docker-compose. Veja [`ajcwebdev/ct3a-docker`](https://github.com/ajcwebdev/ct3a-docker) para um repositório de exemplo baseado neste documento.
+
+## Configuração do Projeto Docker
+
+Por favor, note que o Next.js requer um processo diferente para variáveis de ambiente em _build time_ (disponível no frontend, prefixado por `NEXT_PUBLIC`) e _runtime_, somente do lado do servidor. Nesta demonstração estamos usando duas variáveis. Preste atenção em suas posições no `Dockerfile`, argumentos de linha de comando e `docker-compose.yml`:
+
+- `DATABASE_URL` (usado pelo servidor)
+- `NEXT_PUBLIC_CLIENTVAR` (usado pelo cliente)
+
+### 1. Configuração do Next
+
+Em seu arquivo [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), configure a opção `output` como `standalone` para [reduzir o tamanho de imagens aproveitando automaticamente os rastreamentos de saída](https://nextjs.org/docs/advanced-features/output-file-tracing):
```

Comment:
```suggestion
Em seu arquivo [`next.config.mjs`](https://github.com/t3-oss/create-t3-app/blob/main/cli/template/base/next.config.mjs), configure a opção `output` como `standalone` para [reduzir o tamanho de imagens](https://nextjs.org/docs/advanced-features/output-file-tracing):
```

Over-information is not needed in translations, the target link already explains the process.